utils.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./lib/utils');
```

Comment:
Why re-exporting this module like this exactly ?

Reply:
It’s to explicitely expose this module and enable `require("purs-loader/utils")` instead of `require("purs-loader/lib/utils")` in dependents of this package.